classes/install/Upgrade.inc.php

```diff
@@ -2855,6 +2855,47 @@ function migrateUserAndAuthorNames() {
 		return true;
 	}
 
+	/**
+	* Update assoc_id for assoc_type 531
```

Comment:
One minor thing here: could we add ASSOC_TYPE_SUBMISSION_FILE_COUNTER_OTHER = 531 or so, so that we know all the places that are to be considered if a change for that assoc type is needed.

Reply:
Thanks, is that what you meant?